OWDTestToolkit/DOM/Music.py

```diff
@@ -2,8 +2,8 @@
 
 audio = ('id', 'player-audio')
 done_button = ("xpath", "//*[@id='title-done']")
-song_by_title = ("xpath", "//span[@class='list-main-title'][text()='{}']")
-first_song = ("xpath", '//*[text()="MP4"]')
+song_by_title = ("xpath", '//a[@data-key-range="{}"]')
+first_song = ("xpath", '//a[@data-option="title"]') # Must be xpath
```

Comment:
Actually, it has to be an xpath selector, because the only way to find this element once the music app is opened, is by using getElementByXpath().